.circleci/config.yml

```diff
@@ -152,10 +171,19 @@ workflows:
   version: 2
   test_all:
     jobs:
-      - rspec
-      - minitest
-      - spider
-      - backend
+      - lint
+      - rspec:
+          requires:
+            - lint
+      - minitest:
+          requires:
+            - lint
+      - spider:
+          requires:
+            - lint
+      - backend:
```

Comment:
does really backend requires lint? I mean the lints are only for the fronted, so in my opinion it would be fine to run it as the same time :thinking:

Reply:
Actually interesting point :thinking:

Reply:
`require` *just* means run after, has nothing to do with functional requirements. The purpose of this is to not run anything if there are linting problems (fail earily) to free up the queue.

Reply:
@hennevogel 

> require just means run after, has nothing to do with functional requirements. The purpose of this is to not run anything if there are linting problems (fail earily) to free up the queue.

I know... but if there is for a sample a backend PR (:tada:) why should a problem with rubocop in the frontend (which could have been introduced in another PR) stop their tests? The backend tests take less time than the linters to run....

Reply:
If there is a "*problem*" with rubocop then the PR should not be merged. No matter what it is about. That way a backend PR will never include a rubocop "*problem*" from a frontend PR. Right?

Reply:
@hennevogel 

> If there is a "problem" with rubocop then the PR should not be merged. No matter what it is about. That way a backend PR will never include a rubocop "problem" from a frontend PR. Right?

yes, but you should also not merge a PR with a backend error, and not because of that you run it first. The backend tests are faster than the linters....

Reply:
>  The backend tests are faster than the linters....

Sure but that means you block 2 slots in the queue even if there is a linter problem. This way you only block one.